Cargo.toml

```
@@ -0,0 +1,30 @@
+[package]
+name = "stub"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+heapless = "0.7.10"
+mockall_double = "0.1.0"
+md-5 = { version = "0.10.1", default-features = false }
+[target.'cfg(target_arch = "riscv32")'.dependencies]
+esp32c3-hal = { path = "../esp-hal/esp32c3-hal" }
+esp-hal-common = { path = "../esp-hal/esp-hal-common" }
```

```suggestion
esp32c3-hal = { package = "esp32c3-hal", git = "https://github.com/esp-rs/esp-hal" }
esp-hal-common = { package = "esp-hal-common", git = "https://github.com/esp-rs/esp-hal" }
```

Let's change these to `git` dependencies as well, so others can build without having to replicate your directory structure.

---

Good point !